codecs/jxl/Makefile

```diff
@@ -78,9 +78,14 @@ $(CODEC_MT_SIMD_BUILD_DIR)/Makefile: CXXFLAGS+=-msimd128
 	-B $(@D) \
 	$(<D)
 
-$(CODEC_DIR)/CMakeLists.txt:
+$(CODEC_DIR)/CMakeLists.txt: 
+	rm -rf $(@D)
 	mkdir -p $(@D)
-	git clone $(CODEC_URL) --recursive -j`nproc` --depth 1 --branch $(CODEC_VERSION) $(@D)
+	cd $(@D) && git init .
```

Comment:
`git init $(@D)`

and then you can do `git -C $(@D) remote add ......` so you don't need to do `cd`